android/src/main/java/com/robinpowered/react/vitals/RNVitalsModule.java

```diff
@@ -66,13 +67,39 @@ private WritableMap getMemoryInfo() {
   }
 
   @Override
-  public void onLowMemory() {
+  public void onTrimMemory(level) {
```

Comment:
I'm confused how `level` is valid syntax without a type

Reply:
Good catch! Forgot to copy working code over from Android Studio....